sdk/servicebus/service-bus/test/common/environmentVariables.ts

```diff
@@ -0,0 +1,98 @@
+const isNode = !!process && !!process.version && !!process.versions && !!process.versions.node;
+
+export enum Constants {
```

Comment:
This enum is strongly coupled with the `getEnvVars` function, meaning we don't intend to use this for any other kind of "constants". Therefore, I would recommend to rename these to `TestEnvVars` or `TestEnvVarNames` or something to indicate that these deal with the env variables meant for tests